setup.cfg

```
@@ -41,6 +41,7 @@ package_dir =
     = py
 packages =
     farm_ng
+    farm_ng.api
```

```suggestion
    farm_ng.service
```
api to generic to me indicating the whole thing